src/HttpTransport.cpp

```diff
@@ -10,29 +9,42 @@ using namespace apache::thrift;
 
 namespace cppkin
 {
+
+//#ifdef THRIFT_FOUND
+//    constexpr EncodingTypes::Enumeration _EncodingType = EncodingTypes::Thrift;
+//    constexpr const char* _ContentType = "Content-Type: application/x-thrift";
+//#else
+//    constexpr EncodingTypes::Enumeration _EncodingType = EncodingTypes::Json;
+//    constexpr const char* _ContentType = "Content-Type: application/json";
+//#endif
+
+    constexpr EncodingTypes::Enumeration _EncodingType = EncodingTypes::Json;
+    constexpr const char* _ContentType = "Content-Type: application/json";
+
+
     HttpTransport::HttpTransport() {
     }
 
     HttpTransport::~HttpTransport() {
     }
 
-    void HttpTransport::Submit(const std::vector<Span*> &spans) {
-        EncoderContextThrift context;
+    void HttpTransport::Submit(const std::vector<Span*>& spans) {
 
-        for (auto &span : spans)
-        {
-            Encoder<EncodingTypes::Thrift>::Serialize(context, *span);
-        }
+        EncoderImpl<_EncodingType> encoder;
```

Comment:
Thrift serialization is not going to be supported when using HTTP transportation?
will it not be more appropriate to understand in real time which serialization to use by using the Configuration?